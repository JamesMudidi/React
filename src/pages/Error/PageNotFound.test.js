import React from 'react';
import { shallow } from 'enzyme';
import PageNotFound from 'pages/Error';

describe('PageNotFound', () => {
  it('matches the snapshot', () => {
    const component = shallow(<PageNotFound />);
    expect(component).toMatchSnapshot();
  });
});
