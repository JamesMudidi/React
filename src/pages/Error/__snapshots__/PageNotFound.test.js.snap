// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound matches the snapshot 1`] = `
<_class
  alert-light={true}
  justify-content-sm-center={true}
>
  404
  <br />
   
  The page you have requested for does not exist
</_class>
`;
